src/videoview/LocalVideoView.jsx

```diff
@@ -16,7 +16,7 @@ class LocalVideoView extends React.Component {
 
   componentDidMount = () => {
     const { stream } = this.props;
-    this.video.srcObject = stream.stream;
+    this.video.srcObject = stream;
```

Comment:
I think the component could be mounting before the stream is set. Maybe add a breakpoint here to check?

Reply:
I don't think it is necessary, because if the local stream is invalid, the LocalVideoView component will not be rendered.

https://github.com/pion/ion-app-web/blob/master/src/Conference.jsx#L208